Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2499,8 +2499,11 @@ EventTimingInteractionID LocalDOMWindow::computeInteractionID(Event& event, Even
 
     switch (type) {
     case EventType::keyup: {
-        ASSERT(event.isKeyboardEvent());
-        auto keyboardEvent = downcast<KeyboardEvent>(&event);
+        // Fails on iOS simulated keyboard events:
```

Comment:
Why does this fail on iOS for simulated events? Ditto the other comments indicating failure elsewhere.